.github/workflows/main.yml

```diff
@@ -64,17 +64,17 @@ jobs:
       fail-fast: false
       matrix:
         include: # Run tests on latest version and version used in https://github.com/fivetran/analytics/blob/main/dbt_ft_prod/deployment.yml#L1
-          - dbt-version: 'dbt-bigquery==0.20.2'
+          - dbt-version: 'pip install --force-reinstall MarkupSafe==2.0.1 && pip install dbt-bigquery==0.20.2'
```

Comment:
Can we rename it to something like:
`- install-command: 'pip install --force-reinstall MarkupSafe==2.0.1 && pip install dbt-bigquery==0.20.2'`

Reply:
Renamed to `install-dbt`.